src/index.scss

```diff
@@ -11,6 +11,7 @@
 @import "./Collapsible/Collapsible";
 @import "./CloseButton/CloseButton";
 @import "./Chip/Chip";
+@import "./ChipCarousel/ChipCarousel";
```

Comment:
Curious moment. It seems to me that if the style files are called index, we can remove the duplication of the name from the path here

For example: `@import "./ChipCarousel"`

Reply:
Yeah, I think it can be considered as separate issue